todo.sh

```diff
@@ -1231,7 +1242,7 @@ case $action in
         if [ "${todo:0:2}" != "x " ]; then
             now=$(date '+%Y-%m-%d')
             # remove priority once item is done
-            sed -i.bak "${item}s/^(.) //" "$TODO_FILE"
+            [[ "$TODOTXT_PRESERVE_PRIORITY" = "1" ]] || sed -i.bak $item"s/^(.) //" "$TODO_FILE"
```

Comment:
Those purely cosmetic, I'd prefer to keep the variable `$item` inside the double quotes as `"${item}s/^(.) //"`. My shellcheck isn't clever enough to figure out that the word-split variable contents can safely be kept outside quotes.